### DISCUSSION TITLE
How to save a gif from an animation?

### DISCUSSION THREAD

**Discussion by jalvesz** [kookma/ogpf#47] [Q&A]
Hi, I was doing a small example case for a quick tutorial, and wanted to know if there is a way to save an animation to a gif file?
I published a first version here: https://fortran-lang.discourse.group/t/fortran-code-snippets/2150/58?u=hkvzjal
Now I wanted to change the static images for an animation doing something like this:
```Fortran
  call gp%animation_start(0.0001) ! start animation, set delay
  call gp%axis([-20._wp, 20._wp, -30._wp, 30._wp,0._wp, 50._wp])
  do incr = 0, num_steps, 100
    call gp%lplot(x(1,:incr), x(2,:incr),x(3,:incr))
  end do
  call gp%animation_show()
```
It works, just would like to get the gif to disk if possible.

**Comment by kookma** [comment]
Yes, this is possible, use the gif terminal for GnuPlot http://gnuplot.info/docs_5.5/loc20476.html

but to have a smoother result, you can create a set of png file and create gif in another program.
